source/type/data.cpp

```diff
@@ -63,7 +63,7 @@ namespace navitia { namespace type {
 
 wrong_version::~wrong_version() noexcept {}
 
-const unsigned int Data::data_version = 49; //< *INCREMENT* every time serialized data are modified
+const unsigned int Data::data_version = 50; //< *INCREMENT* every time serialized data are modified
```

Comment:
warning: you might need to re-increment that before merge as it is also incremented in another PR, especially if it is not merged in the 1.31 version.
(the important thing is that it is incremented at least once between 2 navitia version if needed)